docs/sec_av_notation.rst

```diff
@@ -178,8 +178,75 @@ The following are supported:
 
 Analysis metadata
 -----------------
-In addition to metadata, dclab also supports a user-defined analysis
+In addition to metadata, dclab also supports certain analysis
 configuration which is usually part of a specific analysis pipeline
 and thus not considered to be experimental metadata.
 
 .. dclab_config:: calculation
+
+.. _sec_user_meta:
+
+User-defined metadata
+---------------------
+In addition to metadata, dclab also supports the user-defined
+configuration section "user". This section will be stored when
+a file is saved and available when the file is reopened.
+
+**Example**: Setting some "user" metadata
+
+    .. ipython::
+
+        In [1]: import dclab
+
+        In [2]: ds = dclab.new_dataset("data/example.rtdc")
+
+        In [3]: my_metadata = {"inlet": True, "n_channels": 4}
+
+        In [4]: ds.config["user"] = my_metadata
+
+        In [5]: other_metadata = {"outlet": False, "tags": ["track", "RBC"]}
+
+        # we can also add metadata with the `update` method
+        In [6]: ds.config["user"].update(other_metadata)
+
+        # or
+        In [7]: ds.config.update({"user": other_metadata})
+
+        In [8]: print(ds.config["user"])
+
+        # we can clear the "user" section like so:
+        In [9]: ds.config["user"] = {}
+
```

Comment:
Here, please encourage users to use str, bool, float, and int, because containers might not render nicely in Shape-Out or DCOR.